tests/integration_tests.rs

```diff
@@ -209,21 +211,50 @@ fn line_range_multiple() {
         .stdout("line 1\nline 2\nline 4\n");
 }
 
+fn setup_cycle_file(filename: &str, content: &[u8]) -> PathBuf {
+    let path = PathBuf::from(EXAMPLES_DIR).join(filename);
+    let _ = File::create(path.as_path())
+        .expect("Could not open file for writing")
+        .write_all(content)
+        .expect("Failure during write");
+    path
+}
+
 #[test]
 fn basic_io_cycle() {
-    let file_out = Stdio::from(File::open("tests/examples/cycle.txt").unwrap());
+    let filename = setup_cycle_file("cycle1.txt", b"I am not empty");
+
+    let file_out = Stdio::from(File::open(filename).unwrap());
+    let ret = bat_raw_command()
+        .arg("test.txt")
+        .arg("cycle1.txt")
+        .stdout(file_out)
+        .assert();
+    if cfg!(unix) {
+        ret.failure();
+    } else {
+        ret.success();
```

Comment:
Do we really want to *enforce success* on other platforms. Or do we rather want to only  enforce failure on unix? In the latter case, just feature-gate the whole test with `#[cfg(unix)]`.

Reply:
What this test really does is check for the unix behavior, but it could also act as a test that checks that false positives don't happen on other platforms (or teach us about a platform that we missed in the implementation). I would be fine with only running it on unix too, what do you think?

Reply:
I'd prefer running it on unix only